### ISSUE TITLE
Inference issue: All empty detections and tracking_data files

### ISSUE THREAD

**Issue by jackkwok** [VisualComputingInstitute/TrackR-CNN#4]
Hi,

I ran inference on the KITTI test set using the pretrained weights (I didn't train any model). Command line:

`
python main.py configs/conv3d_sep2 "{\"build_networks\":false,\"import_detections\":true,\"task\":\"forward_tracking\",\"dataset\":\"KITTI_segtrack_feed\",\"do_tracking\":true,\"visualize_detections\":true,\"visualize_tracks\":true,\"load_epoch_no\":5,\"video_tags_to_load\":[\"0002\",\"0006\",\"0007\",\"0008\",\"0010\",\"0013\",\"0014\",\"0016\",\"0018\"]}"
`
All the .txt files under ./forwarded/conv3d_sep2/detections/ and ./forwarded/conv3d_sep2/tracking_data/ generated are empty files with 0 bytes.  What am I missing?

Thanks.

**Comment by jackkwok** [comment]
Issue resolved.  I had to modify both the config file and edit DEFAULT_PATH in KITTI_segtrack.py as well to point to the local test directory.

**Comment by thezealousfool** [comment]
@jackkwok I have the same problem. Can you tell me exactly what you did to fix it?
Thanks.

**Comment by GehenHe** [comment]
@jackkwok  Hi, any more details can be provided about the modifications? Getting stuck at this step. 
Much appreciate.
